Dropdown/Dropdown.js

```diff
@@ -319,7 +324,11 @@ const DropdownBase = kind({
 				};
 			});
 		},
-		className: ({css, direction, open, width, title, skin, styler}) => styler.append(`${width}Width`, {hasTitle: Boolean(title), open}, skin === 'silicon' ? classnames(css.dropdownButton, {[css.upDropdownButton]: direction === 'above'}) : {}),
+		className: ({adjusteddirection, css, direction, open, width, title, skin, styler}) => {
```

Comment:
I think we don't need to use both `direction` and `adjusteddirection` to get className. Could you please update this statement to use `adjusteddirection` only?

Reply:
I accepted your suggestion.